src/global/url.ml

```diff
@@ -46,7 +46,9 @@ let minor v =
 
 let v2 = "https://v2.ocaml.org"
 let manual_with_version v = v2 ^ "/releases/" ^ minor v ^ "/htmlman/index.html"
-let manual = "/releases/latest/manual.html"
+let manual = "/manual"
+let manual_lang v = "/manual/" ^ minor v ^ "/index.html"
+let manual_api v = "/manual/" ^ minor v ^ "/api/index.html"
 let api_with_version v = v2 ^ "/releases/" ^ minor v ^ "/api/index.html"
 let api = "/releases/latest/api/index.html"
```

Comment:
```suggestion
let api_with_version v = "/releases/" ^ minor v ^ "/api"
let api = "/releases/latest/api"
```

Reply:
I agree with this change, but I think it's outside the scope of this PR.  A whole exercise will need to be carried out to remove redirects to `v2`.  See `fwd_v2` in `src/ocamlorg_web/lib/redirection.ml`.

Reply:
I agree removing obsolete redirects should be in another PR. But this only allows the links to the Standard Library API (such as found in https://ocaml.org/docs) to point to the local content instead of V2.